README.md

```diff
@@ -718,7 +718,12 @@ considered to be well-supported.
 
 To run the ORT binaries (also see [Installation from binaries](#from-binaries)) at least Java 11 is required. Memory and
 CPU requirements vary depending on the size and type of project(s) to analyze / scan, but the general recommendation is
-to configure Java with 8 GiB of memory (`-Xmx=8g`) and to use a CPU with at least 4 cores.
+to configure Java with 8 GiB of memory and to use a CPU with at least 4 cores.
+
+~~~bash
```

Comment:
Please use backticks an "shell" instead of "bash", as recently aligned on in https://github.com/oss-review-toolkit/ort/pull/7198/commits/23ac766c7b8aedeccad3c069af9a037f17cab516.